frontend/lib/uast/Class.cpp

```diff
@@ -20,15 +20,13 @@
 #include "chpl/uast/Class.h"
 
 #include "chpl/uast/Builder.h"
-#include "chpl/uast/FnCall.h"
-#include "chpl/uast/Identifier.h"
 
 namespace chpl {
 namespace uast {
 
 
 std::string Class::dumpChildLabelInner(int i) const {
-  if (i == parentClassChildNum_) {
+  if (i == inheritExprChildNum_) {
```

Comment:
If `i > inheritExprChildNum` and `i < inheritExprChildNum+numInheritExprs_`, it should print out `implemented interface` IMO